doc/user_guide/impala_user_guide.md

```diff
@@ -106,15 +112,56 @@ Example command:
 
 ```
 python tools/create_kerberos_conn.py krb_conn krbuser@EXAMPLE.COM /etc/krb5.conf ./krbuser.keytab \
-  'jdbc:impala://<Impala host>:<port>;AuthMech=1;KrbRealm=EXAMPLE.COM;KrbHostFQDN=host.example.com;KrbServiceName=impala'
+  'jdbc:impala://<Impala host>:<port>;AuthMech=1;KrbAuthType=1;KrbRealm=EXAMPLE.COM;KrbHostFQDN=host.example.com;KrbServiceName=impala'
 ```
 
 Output:
 
 ```sql
-CREATE CONNECTION krb_conn TO 'jdbc:impala://<Impala host>:<port>;AuthMech=1;KrbRealm=EXAMPLE.COM;KrbHostFQDN=host.example.com;KrbServiceName=impala' USER 'krbuser@EXAMPLE.COM' IDENTIFIED BY 'ExaAuthType=Kerberos;enp6Cg==;YWFhCg=='
+CREATE CONNECTION krb_conn TO 'jdbc:impala://<Impala host>:<port>;AuthMech=1;KrbAuthType=1;KrbRealm=EXAMPLE.COM;KrbHostFQDN=host.example.com;KrbServiceName=impala' USER 'krbuser@EXAMPLE.COM' IDENTIFIED BY 'ExaAuthType=Kerberos;enp6Cg==;YWFhCg=='
+```
+
+#### Using Correct `krb5.conf` Configuration File
+
+The regular Kerberos configuration `/etc/krb5.conf` file may contain `includedir` directives that include additional properties. To accommodate all properties in a single file, please make a copy of `/etc/krb5.conf` file and update it accordingly.
+
+Example of `/etc/krb5.conf` contents:
+
+```
+includedir /etc/krb5.conf.d/
+includedir /var/lib/sss/pubconf/krb5.include.d/
+
+[libdefaults]
+dns_lookup_kdc = false
+dns_lookup_realm = false
+...
+```
+
+Copied and changed `krb5.conf` file after adding properties from `includedir` directories:
+
+```
+# includedir /etc/krb5.conf.d/
+# includedir /var/lib/sss/pubconf/krb5.include.d/
+
+[libdefaults]
+dns_lookup_kdc = false
+dns_lookup_realm = false
+udp_preference_limit = 1
+
+[capaths]
+...
+[plugins]
+...
 ```
 
+The `includedir` folders contain a file with a setting `[libdefaults]` setting `udp_preference_limit = 1`, two new categories `[capaths]` and `[plugins]` that were not in the original `krb5.conf` file. We add such settings into the copied `krb5.conf` file and use it to create the connection object.
+
+This is required since the Virtual Schema run in a sandboxed container that does not have an access to the host operating system's files.
```

Comment:
```suggestion
This is required since the Virtual Schema runs in a sandboxed container that does not have access to the host operating system's files.
```